pkg/runtime/serializer/codec_factory.go

```diff
@@ -44,24 +41,35 @@ type serializerType struct {
 	// be expected to pass into or a gvk to Decode, since no type information will be available on
 	// the object itself.
 	RawSerializer runtime.Serializer
-
 	// Specialize gives the type the opportunity to return a different serializer implementation if
 	// the content type contains alternate operations. Here it is used to implement "pretty" as an
 	// option to application/json, but could also be used to allow serializers to perform type
 	// defaulting or alter output.
 	Specialize func(map[string]string) (runtime.Serializer, bool)
+
+	AcceptStreamContentTypes []string
+	StreamContentType        string
+
+	Framer           runtime.Framer
+	StreamSerializer runtime.Serializer
+	StreamSpecialize func(map[string]string) (runtime.Serializer, bool)
 }
 
 func newSerializersForScheme(scheme *runtime.Scheme, mf json.MetaFactory) []serializerType {
 	jsonSerializer := json.NewSerializer(mf, scheme, runtime.ObjectTyperToTyper(scheme), false)
 	jsonPrettySerializer := json.NewSerializer(mf, scheme, runtime.ObjectTyperToTyper(scheme), true)
 	serializers := []serializerType{
 		{
-			AcceptContentTypes: []string{"application/json"},
+			AcceptContentTypes: []string{"application/json", "application/json;stream=watch"},
```

Comment:
Shouldn't this be the other way, i.e.:
AcceptContentTypes accepts only "application/json"

and AcceptStreamContentTypes accepts both?